src/rtcmix/RTcmixMain.cpp

```diff
@@ -483,6 +534,11 @@ RTcmixMain::interrupt_handler(int signo)
 		interrupt_handler_called = 1;
 	   fprintf(stderr, "\n<<< Caught interrupt signal >>>\n");
 
+           if (udpRcvSocket != NULL){
```

Comment:
I wish we could hide code like this from our top-level RTcmix object - but we can wait to do that until everything else is working.